src/clojure_tictactoe/cli/input/input_getter.clj

```diff
@@ -13,25 +14,38 @@
   (get-user-input)
   nil)
 
-(defn parse-move-input [input]
-  (let [formatted-input (read-string input)]
-    (let [valid-numeral (and
-                          (number? formatted-input)
-                          (< formatted-input 9))]
-      (when valid-numeral
-        formatted-input))))
+(defn parse-move-input [input choice-limit]
+  (let [formatted-input (read-string input)
+        valid-numeral (and
```

Comment:
Can this `let` binding for `valid-numeral` be more descriptive?

Reply:
Do you have a suggestion because I'm actually rather proud of that name.